src/VisualStudio/Core/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -456,32 +472,15 @@ private void Order(List<ClassifiedSpan> syntaxSpans)
                     }
                 }
 
-                return finalParts;
+                return finalParts.ToImmutable();
             }
 
-            private bool ShouldUseSemanticPart(ValueTuple<SymbolDisplayPart, TextSpan> partAndSpan)
+            private bool ShouldUseSemanticPart(ClassifiedSpan partAndSpan)
```

Comment:
Maybe call this IsClassifiedAsText since whether or not it is semantic or syntactic is irrelevant? Note that would invert the logic and the call sites would need to be updated.